{{cookiecutter.project_shortname}}/docker-compose.dev.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Backend services for needed for development.
+#
+# Usage::
+#
+#   $ docker-compose up -d
```

Comment:
This is a good question. The main difference is that prod has the ui/api apps separated and containerized. Instead the other runs the flask debug local.

Which one do you envision being used more? I think the containers is the one that will be used more, and that's why I set it up like this (but ofc it can be changed).

Reply:
I've used `docker-compose.dev.yml` the most, so I made it ``docker-compose.yml` but I was mostly developing. I do like that production needs to be more explicit, so it is an intentional effort from the hoster/developer. ¯\_(ツ)_/¯

Reply:
Deal :) I'll change default to 'dev'. Also in the RFC and the behaviour defaults to `dev`,  Needs file updates in https://github.com/inveniosoftware/rfcs/pull/4/files and docs